tests/protocol/mocks.py

```diff
@@ -209,3 +216,35 @@ def __init__(self, status, body=''):
 
     def read(self, *_):
         return self.body
+
+
+class MockWireClient:
```

Comment:
These 2 classes do not seem to belong here. The mocks here provide generic infrastructure used by multiple tests. These classes provide very limited functionality and don't seem to be useful outside of the tests that are currently using them (MockProtocol doesn't seem to be used at all). I'd suggest moving them to the tests that use them.

Reply:
MockWireProtocol was no longer necessary. MovedMockWireClient to the extensions config retriever tests.